docker/server/push.sh

```diff
@@ -0,0 +1 @@
+docker push gcr.io/oss-forensics-registry/turbinia/turbinia-server:$1
```

Comment:
You can remove this. This will not be done from here, it will be done from within Cloud Build pipeline using Github webhooks.

Reply:
I thought it might make it less error prone while the build pipeline is not setup yet. Removed file.